src/app/utils/helpers.ts

```diff
@@ -0,0 +1,25 @@
+import * as oasis from '@oasisprotocol/client'
+import * as oasisRT from '@oasisprotocol/client-rt'
+
+export const isValidOasisAddress = (addr: string) => {
+  try {
+    oasis.staking.addressFromBech32(addr)
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
+export const isValidEthAddress = (hexAddress: string): boolean => {
+  return /^0x[0-9a-fA-F]{40}$/.test(hexAddress)
```

Comment:
Note: This doesn't validate checksum in address. I'll add a full version for search validation